src/docs/background/how-to-integrate-3rd-party-payment-gateway/index.md

```diff
@@ -0,0 +1,341 @@
+---
+title: How to integrate a 3rd-party payment gateway with your marketplace
+slug: how-to-integrate-3rd-party-payment-gateway
+updated: 2021-03-19
+category: background
+ingress:
+  This guide describes, on a high-level, how to integrate any 3rd-party payment gateway with your Flex-powered marketplace.
+published: true
+---
+
+Sharetribe Flex provides out-of-the-box integration with Stripe. To reach markets not supported by Stripe, relying on another payment gateway can be crucial. This guide describes on a high-level, without going into the details of specific payment gateways, how to integrate any 3rd-party payment gateway (such as [PayPal Commerce Platform](https://www.paypal.com/us/business/platforms-and-marketplaces), [MANGOPAY](https://www.mangopay.com/marketplaces/), or [Adyen for Platforms](https://www.adyen.com/our-solution/marketplaces-and-platforms)) with Sharetribe Flex.
+
+## Prerequisites
+
+Before reading this guide, you should be familiar with the following Sharetribe Flex features:
+
+* [Privileged transitions](https://www.sharetribe.com/docs/background/privileged-transitions/)
+* [Events](https://www.sharetribe.com/docs/references/events/)
+* [Reacting to events](https://www.sharetribe.com/docs/cookbook-events/reacting-to-events/)
+* [Extended data](https://www.sharetribe.com/docs/references/extended-data/)
+
+## Marketplace payment flow
+
+In this section we illustrate a marketplace payment flow in high-level and briefly discuss each stage. Later in this article, we'll go through how to integrate each state with your Flex-powered marketplace.
+
+In a nutshell, a payment flow in a marketplace contains five significant steps. The following diagram illustrates a timeline of these steps:
+
+![alt_text](payment-flow.png "Marketplace payment flow")
```

Comment:
Alt texts are important, especially in documentation!

Reply:
Yes for sure! I'll fix this. I didn't mean to leave alt texts like this, but in fact, I wasn't familiar with the syntax and thought that the "Marketplace payment flow" would be the alt text, but it seems that that becomes the title of the image, not alt text.